src/createHTTPClient.js

```diff
@@ -22,6 +22,13 @@ export default function createHTTPClient(accessToken) {
     },
   });
   return {
+    // General use method to access the underlying
+    // axios client so that calls to specific endpoints
+    // can be made
+    getAxiosClient() {
```

Comment:
Just in case the underlying client switches to `fetch` or something in the future, it might be slightly more generic to just call this `getClient` or something and avoid coupling the method name to the current client implementation.